query/src/sql/exec/mod.rs

```diff
@@ -161,7 +161,7 @@ impl PipelineBuilder {
 
         match plan {
             RelOperator::PhysicalScan(physical_scan) => {
-                self.build_physical_scan(physical_scan, pipeline)
+                self.build_physical_scan(context, physical_scan, pipeline)
```

Comment:
We should also check `build_project`, `build_eval_scalar`, `build_filter`, `build_aggregate`, `build_order_by`, `build_limit`.

Reply:
Fixed.